paper/basic_training.tex

```diff
@@ -1211,10 +1197,51 @@ \subsubsection{Grid based Ewald summation}
 
 
 \end{Checklists*}
+\todo[inline, color={yellow!20}]{DLM: Also need to point out the checklist above and discuss it in the text somewhere.}
+\todo[inline, color={yellow!20}]{DLM: I also need to go over the checklist again and make sure it is what we want/addresses key issues (and everything there is addressed in the text.}
+
+\section{Should you run MD?}
+
+A critical question \emph{before} preparing an MD simulation of your system is whether you even \emph{should} use MD for your system in view of the resources you have and what information you hope to obtain.
+MD is a tool, but it may not be the right tool for your problem.
+Before beginning any study, it is critical to sort out what questions you want to answer, what resources (computational and otherwise) you have at your disposal, and whether you have any information about your system(s) of interest that indicate you can realistically expect to answer those questions given a set of MD simulations.
+
+As noted above, the frequency of the fastest vibrational motions in a system of interest sets a fundamental limit on the timestep which, given fixed computational resources, sets a limit on how much simulation time can be covered with any reasonable amount of computer time.
+Thus, as noted in the Introduction, the longest all-atom MD simulations are on the microsecond to millisecond timescale.
+This means that if your system is complex and answering your questions will require sampling critical events that have a timescale of seconds or longer, MD will not be the right tool for the job.
+You could invest a huge amount of effort running a MD simulations and find that these do nothing whatsoever to address your questions.
+
+Ideally, you should have some information before beginning that the relevant timescales for your system might be accessible given the amount of MD you can afford to run, or you could plan a set of exploratory MD simulations to assess feasibility. 
+But don't simply plunge in and attempt to run simulations until you find the answers to your questions, as the required timescales could end up being orders of magnitude longer than what you can afford to spend.
```

Comment:
I agree with what you say completely, but from here onwards we may be able to condense a little bit and also perhaps adjust the tone.

Reply:
I'll let you attempt to adjust tone in a PR unless you have specific suggestions. Perhaps I wrote slightly informally, though I tend to prefer "we should" (or "us", etc.) to "one should", ESPECIALLY for introductory material, as it has more of a feel of bringing the reader along for the ride. I should probably avoid contractions like "don't" though.